docs/source/manual/usermanual.rst

```diff
@@ -392,97 +492,3 @@ Automation of the verification process
 **************************************
 
 OpenKAT has been created to automate tedious tasks such as this one. We like to include an automated verification process for objects that includes the entire chain of information, with nice green checkmarks. It is on the roadmap, if you want to contribute to it you are most welcome! Get in touch through meedoen@openkat.nl.
-
-What are Config OOI's?
```

Comment:
This was only added last month (cc78ec6ced32967ef594e5fddbe8fae18e8c9b50), so I'd say it deserves a better location rather than being deleted.

Reply:
Most of it is rewritten in this PR: https://github.com/minvws/nl-kat-coordination/pull/1569